aws/v9.3.4/README.md

```diff
@@ -37,6 +52,12 @@ Complete details for changes since v3.10.1 please check the upstream release not
 - Added 100.64.0.0/10 to the allowed egress subnets in NetworkPolicy.
 - Fixed invalid cluster role binding for Helm 3 compatibility.
 
+### Kubernetes 1.16.11 
+- Updated from Kubernetes 1.16.9 - 
+changelog since [v1.16.10](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#changelog-since-v11610)and
```

Comment:
```suggestion
changelog since [v1.16.10](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#changelog-since-v11610) and
```